socketshark/subscription.py

```diff
@@ -28,6 +31,9 @@ def __init__(self, config, session, data):
         # order key -> numeric order (the default order key is None)
         self.order_state = {}
 
+        # throttle key -> (last message sent timestamp, latest message, task)
+        self.throttle_state = {}
```

Comment:
Does this need to be in Redis so throttles are consistent across all SocketShark processes or do we hash sessions to specific SocketShark processes?

Reply:
The throttle is per subscription per session. Nothing needs to be shared since each SocketShark process is directly holding a set of connections (sessions).